deep-learning/_posts/2018-07-26-masklab.md

```diff
@@ -0,0 +1,50 @@
+---
+layout: review
+title:  "MaskLab: Semantic and Direction Features"
+tags:   Segmentation
+author: Faezeh Amjadi
+pdf:    https://arxiv.org/pdf/1712.04837.pdf  
+cite:
+authors: "Liang-Chieh Chen, Alexander Hermans, George Papandreou, Florian Schroff, Peng Wang, Hartwig Adam"
+title:   "
+MaskLab: Instance Segmentation by Refining Object Detection with Semantic and Direction Features"
+venue:   "CVPR 2018"
+---
+
+### Description
+
+The authors present a model, called MaskLab, which produces three outputs:
+box detection, semantic segmentation, and direction prediction.
```

Comment:
Not clear what direction that is.  Is the input image a video?

Reply:
Direction prediction compare a pixel's class to class object's center, if this pixel is belong this object or not.